mastering-plone/thirdparty_behaviors.rst

```diff
@@ -25,17 +25,22 @@ To use it add the name to your list of eggs in :file:`buildout.cfg`:
         ...
         collective.behavior.banner
 
-Even though :py:mod:`collective.behavior.banner` has been released on pypi we'll now act as if this addon exists on github only - or alternatively has changes on github that have not been released yet but you that you really want. This is not to annoy you - it happens surprisingly often if you work with new versions of Plone.
+Even though :py:mod:`collective.behavior.banner` has been released on PyPI we will now act as if this addon exists on GitHub or has changes on GitHub that have not yet been released, but that you really want.
 
```

Comment:
Remove blank line so that it is together with the previous paragraph instead of its own.